crates/sui-replay/src/displays/transaction_displays.rs

```diff
@@ -5,13 +5,23 @@ use crate::displays::Pretty;
 use std::fmt::{Display, Formatter};
 use sui_types::transaction::CallArg::Pure;
 use sui_types::transaction::{
-    write_sep, Argument, CallArg, Command, ObjectArg, ProgrammableMoveCall, ProgrammableTransaction,
+    write_sep, Argument, CallArg, Command, FullPTB, ObjectArg, ProgrammableMoveCall,
+    ProgrammableTransaction,
 };
 use tabled::{
     builder::Builder as TableBuilder,
     settings::{style::HorizontalLine, Panel as TablePanel, Style as TableStyle},
 };
 
+impl<'a> Display for Pretty<'a, FullPTB> {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let Pretty(full_ptb) = self;
+        let (ptb, _results) = full_ptb;
+        // todo: format results and write
```

Comment:
Do we have a VM instance here during replay? 
If so, I think it would be rather handy to be able to show the Move values, rather than just the BCS bytes

Reply:
Agreed. Either that or there's an executor higher up when the FullPTB is getting built, so converting the TypeTag + bytes into a `annotated::Value` (or similar) at that point would then make this part possible without needing to spin up another VM instance.

Reply:
yes, we need the Move values, I think it's pretty critical, so +1 to the above

Reply:
I am working on getting the types resolved to annotated value. In doing so, we need an AuthorityStore in the replay, which we currently don't have access to and can't easily add. I tried some workarounds, but to get the annotated values, it is going to take another large dependency injection design.